manifests/kustomize/overlays/dev/nginx_ingress.yaml

```diff
@@ -0,0 +1,628 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app.kubernetes.io/instance: ingress-nginx
+    app.kubernetes.io/name: ingress-nginx
+  name: ingress-nginx
```

Comment:
For dev this is probably ok, but I'm a little wary of keeping ingres-nginx stuff up to date. Is there an easy, one-line way to update all this if ingress-nginx does a new release?

Reply:
I was just scouting around for the same reason. Perhaps their helm chart could be of use. 

```
kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.2.0/deploy/static/provider/cloud/deploy.yaml

https://kubernetes.github.io/ingress-nginx/deploy/#quick-start```

Reply:
sure, updated.